src/components/GDPRForm/index.tsx

```diff
@@ -5,11 +5,11 @@ export const GDPRForm = () => (
     <>
         <br />
         <p className="px-4 font-bold text-center z-10 relative mb-0 text-sm md:text-lg">
-            Want PostHog hosted on an EU Cloud?
+            Interested in PostHog Cloud EU?
         </p>
         <div className="flex flex-col md:flex-row justify-center items-center gap-2 xl:gap-4">
-            <CallToAction width="106" to="/signup/eu-cloud">
-                Join the waitlist!
+            <CallToAction width="106" to="/eu">
+                Signup now!
```

Comment:
```suggestion
                Try PostHog Cloud EU
```

Reply:
Considered putting 'for free' or something like that here? Yes or no?

Reply:
I vote yes